filebeat/tests/system/test_input.py

```diff
@@ -14,6 +15,73 @@
 
 
 class Test(BaseTest):
+    def test_fixup_registry_entries_with_global_id(self):
+        """
+        Should update the registry correctly when there were entries using the
+        default '.global' input ID
+        """
+        testfile = os.path.join(self.working_dir, "log", "test.log")
+        self.render_config_template(
+            template_name="filebeat-duplicated-id",
+            path=testfile,
+        )
+        os.mkdir(os.path.join(self.working_dir, "log/"))
+        file = open(testfile, 'w')
+        offset = 0  # keep count of the bytes written
+        for n in range(0, 5):
+            offset += file.write("hello world\n")
+        file.close()
```

Comment:
```suggestion
        with open(testfile, 'w') as file:
           offset = 0  # keep count of the bytes written
           for n in range(0, 5):
               offset += file.write("hello world\n")
```

Using `with` will automatically close the file when exiting the scope of the `with`.